Makefile

```diff
@@ -70,6 +69,16 @@ endif
 TARGET_ARCH = $(HOST_ARCH)
 
 # ---------------------------------------------------------------------
+# Code signing
+# ---------------------------------------------------------------------
+
+ifeq ($(TARGET_PLATFORM),darwin)
+ifndef CODESIGN_IDENTITY
+$(warning No code-sign identity found (CODESIGN_IDENTITY))
```

Comment:
Done! I've clarified the message and renamed `CODESIGN_IDENTITY` to `CODE_SIGN_IDENTITY`.